cola/widgets/prefs.py

```diff
@@ -140,6 +142,12 @@ def __init__(self, context, model, parent, source):
         self.show_path = qtutils.checkbox(checked=True)
         self.tabwidth = standard.SpinBox(value=8, maxi=42)
         self.textwidth = standard.SpinBox(value=72, maxi=150)
+        self.logdate = QtWidgets.QLineEdit()
+        tooltip = N_('Similar to git log --date option')
+        self.logdate.setToolTip(tooltip)
+        regex = QtCore.QRegExp(logdate.create_regex())
+        validator = QtGui.QRegExpValidator(regex, self.logdate)
+        self.logdate.setValidator(validator)
```

Comment:
About the combo, actually at first I did use `qtutils.combo()` and then, as you mentioned, I changed it to text box just because of the `format:...` option. After the changes, the combo box is not editable, because I think that's not necessary. As a result, I removed the validator as well.